src/commands/classified/external.rs

```diff
@@ -254,7 +254,7 @@ async fn spawn(
         process = process.arg(&arg);
     }
 
-    process = process.cwd(path);
```

Comment:
This sets the current working directory for the process. I think you want to tweak line 251 to be `Exec::shell`?

Reply:
Good catch

Reply:
That's what I get for doing the edit when I'm not paying attention :)